R/aes-calculated.r

```diff
@@ -54,6 +60,14 @@ is_calculated <- function(x) {
     stop("Unknown input:", class(x)[1])
   }
 }
+is_modifier <- function(x) {
+  if (!is.call(x)) return(FALSE)
```

Comment:
Can you please rewrite to use the same branches as above? It's a pattern that make it clear that you're recursing over an expression tree

Reply:
Sure... The reason I didn't was because `is_calculated()` seemed to only be this complex because it had to deal with the old dotted notation